app/src/main/java/org/p2p/wallet/home/ui/crypto/MyCryptoPresenter.kt

```diff
@@ -87,7 +87,7 @@ class MyCryptoPresenter(
                 // pnl must restart it's 5 minutes timer after force refresh
                 // and we should restart it only if it was initially started
                 // other cases might indicate that we didn't start observer due to empty state
-                if (pnlInteractor.isStarted()) {
+                if (!pnlInteractor.canBeStarted()) {
```

Comment:
it looks there are logical mistake: `if it can't be started - restart`
I suppose you wanted to check whether it can be started at all, considering feature toggle condition, in this case you need to add one more `if`:
```
if(pnlIntereactor.canBeStarted() && pnlInteractor.isStarted()) {
    pnlInteractor.restartAndRefresh()
}
```

checking for "isStarted" is necessary here, because we should **restart** only if it was initially started after the first tokens load (see `handleTokenState` `is UserTokensState.Loaded` block)